packages/network/src/v2/ecsEventFromLog.ts

```diff
@@ -19,7 +19,7 @@ export const ecsEventFromLog = async (
   const tableId = TableId.fromBytes32(utils.arrayify(args.table));
   const component = tableId.toString();
   // TODO: revisit key tuple encoding for client
-  const entity = args.key.join(":") as EntityID;
+  const entity = `v2:${args.key.join(":")}` as EntityID;
```

Comment:
This might break something if I want to use v2 with ECS, and want to refer to another entity in my component

Example: `OwnedBy` component, Bob (0x01) is owned by Alice (0x02), so I have an entry in the OwnedBy table with key `0x01` and value `0x02`. Now I want to do an recs query on the client to get all entities owned by Alice, so I do `getEntitiesWithValue(OwnedBy, {value: Alice})`, but because Alice is normalized to `v2:0x02` it wouldn't find the `0x02` value of key `0x01`.

Reply:
updated in https://github.com/latticexyz/mud/pull/415/commits/9c13cd7084ed8afb0754a0eefbeb756f9f38e375